### PR TITLE
[Docs] introduce "make help"

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Comment by sapk** [comment]
What about a more generic way like https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html or https://stackoverflow.com/a/35730928 ?

**Review by sapk — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -92,6 +92,24 @@ all: build
...
+.PHONY: help
+help:
+	@echo "Make Routines:"
+	@echo " - \"\"                equivalent to \"build\""
+	@echo " - build             creates the entire project"
+	@echo " - clean             delete integration files and build files but not css and js files"
+	@echo " - clean-all         delete all generated files (integration test, build, css and js files)"
+	@echo " - css               rebuild only css files"
+	@echo " - js                rebuild only js files"
+	@echo " - generate          equivalent to \"css js\""
```
Comment (techknowlogick) [inline Makefile]: also calls, `go generate...`

**Review by techknowlogick — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (6543) [inline Makefile]:
```suggestion
	@echo " - generate          run \"make css js\" and \"go generate\""
```

**Comment by 6543** [comment]
@techknowlogick done

File: Makefile [hunk 2]
```diff
@@ -92,6 +92,24 @@ all: build
...
+	@echo " - build             creates the entire project"
+	@echo " - clean             delete integration files and build files but not css and js files"
+	@echo " - clean-all         delete all generated files (integration test, build, css and js files)"
+	@echo " - css               rebuild only css files"
+	@echo " - js                rebuild only js files"
+	@echo " - generate          run \"make css js\" and \"go generate\""
+	@echo " - fmt               format the code"
+	@echo " - generate-swagger  generate the swagger spec from code comments"
+	@echo " - swagger-validate  check if the swagger spec is valide"
+	@echo " - revive            run code linter revive"
```
Comment (davidsvantesson) [inline Makefile:108]: add vet

Comment (6543) [reply Makefile:108]: @davidsvantesson dont know what **vet**'s function is

Comment (6543) [reply Makefile:108]: ... Vet examines Go source code and reports suspicious constructs ...

OK

Comment (6543) [reply Makefile:108]: done

**Review by davidsvantesson — APPROVED** [review]

**Comment by 6543** [comment]
@techknowlogick you still block this - are you ok with this?

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**